packages/serverless/src/backend_connector.js

```diff
@@ -510,6 +489,27 @@ function onTimeout(localUseLambdaExtension, req, resourcePath, payload, finalLam
   }
 }
 
+function cleanupRequest(req) {
+  // When the Node.js process is frozen while the request is pending, and then thawed later,
+  // this can trigger a stale, bogus timeout event (because from the perspective of the freshly thawed Node.js
+  // runtime, the request has been pending and inactive since a long time).
+  // To avoid that, we remove all listeners
+  // (including the timeout listener) on the request. Since the Lambda runtime will be frozen afterwards (or
+  // reused for a different, unrelated invocation), it is safe to assume that
+  // we are no longer interested in any
+  // events emitted by the request or the underlying socket.
+  req.removeAllListeners();
+
+  // We need to have a listener for errors that ignores everything, otherwise aborting the request/socket will
+  // produce an "Unhandled 'error' event"
+  req.once('error', () => {});
+
+  // Finally, abort the request because from our end we are no longer interested in the response and we also do
+  // not want to let pending IO actions linger in the event loop. This will also call request.destoy and
```

Comment:
```suggestion
  // not want to let pending IO actions linger in the event loop. This will also call request.destroy and
```
One minor suggestion: There's a typo in the comment.